backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlSchemaMetadata.java

```diff
@@ -139,14 +139,16 @@ public SchemaMetadata create(TableMetadata... tables) {
     return this;
   }
 
-  private static void validateTableIdentifierIsUnique(SqlSchemaMetadata sm, TableMetadata table) {
+  private static void validateNameIsLowercaseUnique(SqlSchemaMetadata sm, TableMetadata table) {
     for (TableMetadata existingTable : sm.getTables()) {
       if (!existingTable.getTableName().equals(table.getTableName())
-          && existingTable.getIdentifier().equals(table.getIdentifier())) {
+          && existingTable.getTableName().equalsIgnoreCase(table.getTableName())) {
         throw new MolgenisException(
             String.format(
-                "Cannot create/alter because name resolves to same identifier: '%s' has same identifier as '%s' (both resolve to identifier '%s')",
-                table.getTableName(), existingTable.getTableName(), table.getIdentifier()));
+                "Cannot create/alter because name resolves to same lowercase: '%s' is same '%s' (equals '%s')",
+                table.getTableName(),
+                existingTable.getTableName(),
+                table.getTableName().toLowerCase()));
       }
     }
   }
```

Comment:
Same, duplication checking should be seperate PR

Reply:
No, this function already existed for identifier so needed to be refactored.